proto/regen/ecocredit/v1/state.proto

```diff
@@ -195,6 +195,9 @@ message BatchBalance {
 
     // retired is the retired amount of credits
     string retired = 4;
+
+    // escrowed is the amount of credits locked up in escrow for the marketplace.
+    string escrowed = 5;
```

Comment:
Is escrowed there right user facing name? Maybe `in_sell_orders`?

Reply:
is there a specific concern you have regarding users and the word `escrowed`? i felt like it was a good fit - `in_sell_orders` seems a bit verbose

Reply:
I could go either way. `in_sell_orders` is more specific and potentially more user friendly but provided the supporting documentation, `escrowed` seems ok to me as well.

Reply:
How about you make a call @ryanchristo

Reply:
Ok, `escrowed` it is. More aligned with `tradable` and `retired`.